src/app.tsx

```diff
@@ -168,13 +169,19 @@ export const App = () => {
 				<Route element={<Navigate replace to="/404" />} path="*" />
 			</Route>
 
-			<Route element={<SettingsLayout />} path="settings">
+			<Route element={<SettingsLayout variant="userSettings" />} path="settings">
 				<Route element={<Profile />} index />
 				<Route element={<ClientConfiguration />} path="client-configuration" />
 
 				<Route element={<Navigate replace to="/404" />} path="*" />
 			</Route>
 
+			<Route element={<SettingsLayout variant="organization" />} path="organization-settings">
+				<Route element={<NewOrganization />} path="new" />
+
+				<Route element={<Navigate replace to="/404" />} path="*" />
+			</Route>
```

Comment:
Check if you can use here `<Outlet/>` instead of variant, and then you will pass the menu as a separate route, for example:
HTTP://localhost:8000/settings/personal - will contain the user-menu component with the user-menu items array.
and then
HTTP://localhost:8000/settings/organization - will contain the org-menu with org-menu items.

Reply:
discussed